tetragonpod/api/v1alpha1/client/register_crd.go

```diff
@@ -0,0 +1,83 @@
+package client
```

Comment:
let's move the logic here to https://github.com/cilium/tetragon/blob/main/pkg/k8s/apis/cilium.io/client/register_v1_crd.go.

Reply:
yes I am working on that logic, I just pushed initial commits.

Reply:
done